tests/api/v1/botActions/deleteWithoutPerms.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * tests/api/v1/botActions/deleteWithoutPerms.js
+ */
+'use strict';
+const supertest = require('supertest');
+const api = supertest(require('../../../../index').app);
+const constants = require('../../../../api/v1/constants');
+const u = require('./utils');
+const tu = require('../../../testUtils');
+const Op = require('sequelize').Op;
+const r = require('../rooms/utils');
+const rt = require('../roomTypes/utils');
+const b = require('../bots/utils');
+const Room = tu.db.Room;
+const User = tu.db.User;
+const RoomType = tu.db.RoomType;
+const Bot = tu.db.Bot;
+const BotAction = tu.db.BotAction;
+const path = '/v1/botActions';
+const expect = require('chai').expect;
+const pfx = '___';
+
+describe('tests/api/v1/botActions/deleteWithoutPerms.js >', () => {
+  let testBotAction;
+  let validToken;
+  let invalidToken;
+  let user;
+
+  before((done) => {
+    tu.createToken()
```

Comment:
I don't think you need this as you are creating a token below for the user. This token is never used